src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -177,21 +208,28 @@ Set `P` to be a square.
 julia> square = cube(2)
 A polyhedron in ambient dimension 2
 
-julia> ntv = NormalToricVariety(square)
+julia> ntv = normal_toric_variety(square)
 A normal toric variety
 ```
 """
-function NormalToricVariety(P::Polyhedron; set_attributes::Bool = true)
-    variety = NormalToricVariety(normal_fan(P))
+function normal_toric_variety(P::Polyhedron; set_attributes::Bool = true)
+    variety = normal_toric_variety(normal_fan(P))
     if set_attributes
         set_attribute!(variety, :polyhedron, P)
     end
     return variety
 end
 
 
+function NormalToricVariety(P::Polyhedron; set_attributes::Bool = true)
+    Base.depwarn("'NormalToricVariety(P::Polyhedron; set_attributes::Bool = true)' is deprecated, use"*
```

Comment:
Maybe better to move these deprecations to `src/Deprecations.jl`, together with a comment on the version (look at the existing content of the file to see what I mean). That way, it is easier for us to remove it in the future.

Reply:
Done. I updated the PR just now.